File: 0001-two-sum/0001-two-sum.ts
function twoSum(nums: number[], target: number): number[] {
    const hash = {};
    for(let i = 0; i < nums.length; i++){
        const num = nums[i];
        if(hash[num] !== undefined)return [hash[num], i];
        hash[target-num] = i;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hash = {}
        for ind, num in enumerate(nums):
            if num in hash:
                return hash[num],ind
            else: 
                hash[target - num] = ind
         